packages/@react-spectrum/table/src/table.css

```diff
@@ -39,3 +39,11 @@
   width: 100%;
   height: 100%;
 }
+
+.react-spectrum-Table-cell--hideHeader {
+  padding: 0;
+}
+
+.react-spectrum-Table-cellContents--hideHeader {
+  text-overflow: clip;
```

Comment:
curious, why is clip needed?

Reply:
Here I am overriding `text-overflow: ellipsis` with default in https://github.com/adobe/react-spectrum/blob/main/packages/@adobe/spectrum-css-temp/components/table/index.css#L156 as we wouldn't want to add ellipsis hideHeader column

Reply:
does it affect something if we don't touch it? we wouldn't render anything in there right?

Reply:
This is the hideHeader column content, so we would have the action button in there. 
In absence of this style, i see ellipsis getting added when a `showDivider` props is added to the column. I am going to make some changes to the styles of hideHeader cell and contents and the defaultWidth so we don't need to default this